frontend/src/metabase/query_builder/components/notebook/steps/DataStep/DataStep.unit.spec.tsx

```diff
@@ -106,6 +109,9 @@ describe("DataStep", () => {
   });
 
   afterAll(() => {
+    HTMLElement.prototype.scrollBy = scrollBy;
+    HTMLElement.prototype.getBoundingClientRect = getBoundingClientRect;
```

Comment:
Cleaning up the mocks, addressing feedback from previous PR: https://github.com/metabase/metabase/pull/42565/files#r1604585537